# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../akonadi/fix-shared-files.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- akonadi-1.0.0/CMakeLists.txt.vanilla	2008-09-07 18:20:31.000000000 +1000
+++ akonadi-1.0.0/CMakeLists.txt	2008-09-07 18:20:49.000000000 +1000
@@ -123,7 +123,7 @@
 ############# install stuff ######################
 
 install(FILES akonadi-mime.xml DESTINATION ${XDG_MIME_INSTALL_DIR})
-update_xdg_mimetypes(${XDG_MIME_INSTALL_DIR})
+#update_xdg_mimetypes(${XDG_MIME_INSTALL_DIR})
 
 if(NOT WIN32)   # pkgconfig file
    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/akonadi.pc.cmake ${CMAKE_CURRENT_BINARY_DIR}/akonadi.pc @ONLY)
